scripts/multisigTransactionProposals/README.md

```diff
@@ -0,0 +1,85 @@
+# Proposing Multisig Transactions via the Safe SDK
+
+The [Safe SDK](https://github.com/safe-global/safe-core-sdk/blob/main/guides/integrating-the-safe-core-sdk.md) is being
+used to propose transactions that only the Safe multisig admin wallet can perform. The proposer of these transactions is
+also part of the multisig wallet but is used to orchestrate the process. E.g. If you require 3 of 5 trusted signers to
+sign a transaction, then the multisig wallet should require 4 of 5 signers where there 4th signer is the orchestrator
+wallet.
+
+## Set Permissioned Prover
+
+To enable the permissioned prover on the light client contract, ensure that the following environment variables are set
+in the `.env` file:
+
+- `SEPOLIA_RPC_URL`
+- `ORCHESTRATOR_SIGNER_KEY`
```

Comment:
Since hotshot also has an Orchestrator it would be good to add something to this name to distinguish it.